smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -85,21 +92,21 @@ function displayReaderList(readers) {
     goog.asserts.assert(readersListElement);
 
     var indicatorClasses = 'reader-state-indicator reader-state-indicator-' +
-                           reader['status'];
+                            reader['status'];
     if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.SUCCESS &&
         reader['isCardPresent']) {
       indicatorClasses = 'reader-card-present-indicator';
     }
     var indicator = goog.dom.createDom('span', indicatorClasses);
 
     var indicatorContainer = goog.dom.createDom(
-        'span', 'reader-indicator-container', indicator);
+      'span', 'reader-indicator-container', indicator);
 
     var text = makeReaderNameForDisplaying(reader['name']) +
-        (reader['error'] ? ' (Error ' + reader['error'] + ')' : '');
+      (reader['error'] ? ' (Error ' + reader['error'] + ')' : '');
 
     var element = goog.dom.createDom('li', undefined, indicatorContainer, text);
-    goog.dom.append(readersListElement, element);
+      goog.dom.append(readersListElement, element);
```

Comment:
nit: Ditto for unwanted indentation change - these two extra spaces aren't needed.